GothicVRProject/GothicVR
# add(DxMusicv2): Implemented DxMusic with synthesizer and on-demand tr…

…ansitions.

# To test
0. Please build your game for MVR to test if the new DLLs run there as well!
1. Start game
2. Is the music the right one when you spawn?
3. Walk around and listen the audio transitions from start area, world, OC, PSI, ...
4. Start lab
5. To your right, there's music handler. Start a few _std.sgt tracks and listen to transitions


Boundaries:
* When loading new game from menu, the async loading bar isn't visible and music pauses. It's another bug unrelated to music change.
* If you're using XRDeviceSimulator, you'll be spawned a few centimeters too low for the ow_ban_day theme. If you walk 10cm forwards, you'll meet it again. This is a "bug" caused by incorrect height of XRDeviceSimulator only. The music plays the right tune from start if you go in with VR glasses at a 1.60+m height as normal.
* In Lab scene, test the _std elements only. There's no fgt (fight) etc. implemented so far.
* Check the reverb value (the hall effect. Most noticeable when you pause the game). I'm not sure if I picked the right value. Please compare with original G1.

# Checklist
(Please remove entries which aren't needed for your PR)

## Scene

# Testing
